modules/datadog/Makefile

```diff
@@ -19,14 +20,46 @@ define datadog_notify
 	@envsubst < $(MAKEFILE_DIR)/modules/datadog/$(1).json | $(CURL)
 endef
 
-## Notify datadog a deploy is starting
+## Notify datadog of a deploy is starting
 datadog\:notify-deploy-starting:
-	$(call datadog_notify,$(subst datadog:notify-,,$@))
+	$(call datadog_notify_deploy,$(subst datadog:notify-,,$@))
 
-## datadog datadog a deploy was successful
+## Notify datadog of a deploy was successful
 datadog\:notify-deploy-success:
-	$(call datadog_notify,$(subst datadog:notify-,,$@))
+	$(call datadog_notify_deploy,$(subst datadog:notify-,,$@))
 
-## datadog datadog a deploy failure
+## Notify datadog of a deploy failure
 datadog\:notify-deploy-failure:
-	$(call datadog_notify,$(subst datadog:notify-,,$@))
+	$(call datadog_notify_deploy,$(subst datadog:notify-,,$@))
+
+define datadog_notify_release
```

Comment:
Not averse to moving it to the `release-harness`, however, it's hardcoded in the `kubernetes:deploy` target to use it. Moving it to the `release-harness` would not be backwards compatible. 

![image](https://cloud.githubusercontent.com/assets/52489/16675025/6393aa6a-4472-11e6-9386-d6c41467b94a.png)